crates/core/src/surfnet/locker.rs

```diff
@@ -504,6 +469,18 @@ impl SurfnetSvmLocker {
 
         Ok(results)
     }
+
+    pub fn account_to_rpc_keyed_account<T: ReadableAccount + Send + Sync>(
+        &self,
+        pubkey: &Pubkey,
+        account: &T,
+        config: &RpcAccountInfoConfig,
+        token_mint: Option<Pubkey>,
+    ) -> RpcKeyedAccount {
```

Comment:
I'm not a huge fan of having / re-inforcing inter-dependencies between surfnets and the RPC layer